### PR TITLE
DM-33521: Split plugins out of ctrl_bps

### PR DESCRIPTION
**Pull request by mxk62**

### CODE REVIEW

File: COPYRIGHT [hunk 1]
```diff
@@ -0,0 +1 @@
+Copyright 2022 University of Illinois Board of Trustees
```
Comment (timj) [inline COPYRIGHT:1]: Presumably there are some more years to add to this?

File: .pre-commit-config.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v2.3.0
+    hooks:
+      - id: check-yaml
+        args:
+          - "--unsafe"
+      - id: end-of-file-fixer
+      - id: trailing-whitespace
+  - repo: https://github.com/psf/black
+    rev: 22.1.0
```
Comment (timj) [inline .pre-commit-config.yaml:11]: 22.3.0

File: examples/SConscript [hunk 3]
```diff
@@ -0,0 +1,3 @@
+# -*- python -*-
```
Comment (timj) [inline examples/SConscript:1]: Delete if not examples.

File: pyproject.toml [hunk 4]
```diff
@@ -0,0 +1,46 @@
+[tool.towncrier]
+    package = "lsst.ctrl.bps"
```
Comment (timj) [inline pyproject.toml:2]: Fix name here and below.

**Review by timj — APPROVED** [review]
Looks fine. Minor comments that are the same as panda. I assume they might also be relevant for the pegasus one.

File: doc/lsst.ctrl.bps.htcondor/userguide.rst [hunk 5]
```diff
@@ -0,0 +1,185 @@
+.. _htc-plugin-overview:
+
+Overview
+--------
+
+LSST Batch Processing Service (BPS) allow large-scale workflows to execute in
```
Comment (MichelleGower) [inline doc/lsst.ctrl.bps.htcondor/userguide.rst:6]: Typo: allows

File: doc/lsst.ctrl.bps.htcondor/userguide.rst [hunk 6]
```diff
@@ -0,0 +1,185 @@
...
+well-managed fashion, potentially in multiple environments.  The service is
+provided by `ctrl_bps`_ package.  ``ctrl_bps_htcondor`` is a plugin allowing
```
Comment (MichelleGower) [inline doc/lsst.ctrl.bps.htcondor/userguide.rst:8]: by the

File: COPYRIGHT (hunk 1 above)
Comment (mxk62) [reply COPYRIGHT]: Indeed, fixed.

File: doc/lsst.ctrl.bps.htcondor/userguide.rst [hunk 7]
```diff
@@ -0,0 +1,185 @@
...
+
+Checking status
+---------------
+
+See `bps report`_.
+
+.. Describe any plugin specific ascpects of a checking submission status below
+   if any.
+
+Occasionally, some jobs are put on hold by HTCondor.  To see the reason why a
```
Comment (MichelleGower) [inline doc/lsst.ctrl.bps.htcondor/userguide.rst:114]: "a jobs are"  either all plural or all singular

**Review by MichelleGower — APPROVED** [review]
I ran a test workflow with this on the NCSA cluster and it ran successfully.  Some minor questions/comments mostly about the documentation.

File: doc/lsst.ctrl.bps.htcondor/index.rst [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+
+   userguide.rst
+
+.. _lsst.ctrl.bps.htcondor-contributing:
+
+Contributing
+============
+
+``lsst.ctrl.bps.htcondor`` is developed at https://github.com/lsst/ctrl_bps_htcondor.
+You can find Jira issues for this module under the `ctrl_bps_htcondor <https://jira.lsstcorp.org/issues/?jql=project%20%3D%20DM%20AND%20component%20%3D%20ctrl_bps_htcondor>`_ component.
```
[the remark this answers is not in the document]
Comment (timj) [reply doc/lsst.ctrl.bps.htcondor/index.rst:37]: I will make them.

**Pull request merged**